src/rotest/cli/client.py

```diff
@@ -178,6 +184,17 @@ def main(*tests):
     if len(tests) == 0:
         tests = discover_tests_under_paths(options.paths)
 
+    if options.filter not in (None, "*"):
+        new_tests = []
+        for test in tests:
+            test_tags = get_tags_by_class(test)
```

Comment:
we lose the ability to filter by test_method_name
which is fine by me, just make sure it's fine by the rest